v3/util/iam-proxy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+source /etc/environment
+
+if [ "${NODE_ROLE}" != "worker" ]; then
+    exit 0
+fi
+
+etcdctl set /images/iam-proxy  	 "behance/iam-docker:latest"
+docker pull $(etcdctl get /images/iam-proxy)
+
+export NETWORK="bridge"
+export GATEWAY="$(ifconfig docker0 | grep "inet " | awk -F: '{print $1}' | awk '{print $2}')"
+export INTERFACE="docker0"
+
+# These will not work until Docker > 1.9 is running
+#export GATEWAY="$(docker network inspect "$NETWORK" | grep Gateway | cut -d '"' -f 4)"
+#export INTERFACE="br-$(docker network inspect "$NETWORK" | grep Id | cut -d '"' -f 4 | head -c 12)"
+
+sudo iptables -t nat -I PREROUTING -p tcp -d 169.254.169.254 --dport 80 -j DNAT --to-destination "$GATEWAY":8080 -i "$INTERFACE"
+
+# Sometimes this is needed?
+systemctl daemon-reload
```

Comment:
@chr0n1x maybe? https://github.com/adobe-platform/mesos-systemd/blob/master/v3/util/etcd2-setup.sh#L20
We should change in both places if so, this was just a copy pasta